docker-compose.yaml

```diff
@@ -1,25 +1,35 @@
 ## Docker Compose File for Rubin-LSST DRP Messaging Stack
 
+# Including the docker compose from monitoring to run monitoring services
+include:
```

Comment:
I like this as a way to separate out various aspects into different compose files but have a master one to deploy from